go/pkg/counter/redis_test.go

```
@@ -443,3 +444,490 @@ func TestRedisCounterMultiGet(t *testing.T) {
+		close(startBarrier)
+		wg.Wait()
+		close(results)
+		
+		// Analyze results
+		var successCount, failureCount int
+		var successfulRemainingValues []int64
+		
+		for res := range results {
+			if res.success {
+				successCount++
+				successfulRemainingValues = append(successfulRemainingValues, res.remaining)
```

golangci-lint (2.2.2)

SA4010: this result of append is never used, except maybe in other appends

(staticcheck)